fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -1146,6 +1147,24 @@ public List<List<Comparable>> getAlterJobInfosByDb(Database db) {
         return rollupJobInfos;
     }
 
+    public List<List<Comparable>> getAllAlterJobInfos() {
```

Comment:
Same as AlterHandler, no need to check priv

Reply:
fixed